pom.xml

```diff
@@ -0,0 +1,138 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>rayfowler</groupId>
+    <artifactId>rotp-public</artifactId>
+    <version>1.0</version>
```

Comment:
Since `jpackage` is a JDK native binary, it assumes strict adherence to Java versioning schemas (see JEP-223: https://openjdk.java.net/jeps/223). That is, $MAJOR.$MINOR.$SECURITY where each var must be a number.  This factors into the _name_ of the resultant installer, but not into the name of the binary itself.  That means this _could_ be used by Rotp.java but can also be a completely independent version number.